### PR TITLE
release cjs esm umd version of sdk, add travis ci config

### PR DESCRIPTION
**Pull request by yqrashawn**
----

#

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/js-conflux-sdk/5)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /dist
 /node_modules
 /coverage/
+/esm/
+/lib/
```
Comment (GeekBerry) [inline .gitignore]: \n ?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "js-conflux-sdk",
+  "name": "@yqrashawn/js-conflux-sdk",
```
Comment (GeekBerry) [inline package.json]: @yqrashawn prefix

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
   "description": "JavaScript Conflux Software Development Kit",
-  "version": "0.7.4",
+  "version": "0.7.4-c",
```
Comment (GeekBerry) [inline package.json]: what mean of '-c'

File: .gitignore (hunk 1 above)
Comment (yqrashawn) [reply .gitignore]: Didn't get what you mean.

**Comment by resodo** [comment]
___
*[README.md, line 31 at r3](https://reviewable.io/reviews/conflux-chain/js-conflux-sdk/5#-M-wN3cH9XkNXqYWtH1u:-M-wN3cIEaWlv6IQVf0K:b-50tc3h) ([raw file](https://github.com/conflux-chain/js-conflux-sdk/blob/5ecbf16256f5a6a0cd4d39a8c77f887721db459a/README.md#L31)):*
> ```Markdown
> main();
> ```
> require neseted file/dir  
> ```

nested?

**Comment by resodo** [comment]
___
*[development/test-create-react-app-default-bundler.sh, line 8 at r3](https://reviewable.io/reviews/conflux-chain/js-conflux-sdk/5#-M-wNo0hCjMWmGUUbYn1:-M-wNo0hCjMWmGUUbYn2:b-fv2ssy) ([raw file](https://github.com/conflux-chain/js-conflux-sdk/blob/5ecbf16256f5a6a0cd4d39a8c77f887721db459a/development/test-create-react-app-default-bundler.sh#L8)):*
> ```Shell
> cd test-sdk-frontend-import-bundler
> sed -i.old "1s;^;import { Conflux } from '../../'" src/App.js
> yarn build
> ```

ended with extra \n?

**Comment by resodo** [comment]
___
*[development/test-create-react-app-default-bundler.sh, line 8 at r3](https://reviewable.io/reviews/conflux-chain/js-conflux-sdk/5#-M-wNu-93D_yDjyRcR3J:-M-wNu-93D_yDjyRcR3K:bnj7hl0) ([raw file](https://github.com/conflux-chain/js-conflux-sdk/blob/5ecbf16256f5a6a0cd4d39a8c77f887721db459a/development/test-create-react-app-default-bundler.sh#L8)):*
> ```Shell
> cd test-sdk-frontend-import-bundler
> sed -i.old "1s;^;import { Conflux } from '../../'" src/App.js
> yarn build
> ```

same as above

**Comment by yqrashawn** [comment]
Just fixed problems above

**Pull request merged**